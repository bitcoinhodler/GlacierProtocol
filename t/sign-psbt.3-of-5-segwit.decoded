{
  "txid": "fed969d84b56aa7e33c92e615984a93ad7ddd930da3d621b4b7eabccaf9c0aa6",
  "hash": "63b0707342f7eff2dd5e16b5d68331bec269853b810407f3af383ec0ceb3d353",
  "version": 2,
  "size": 514,
  "vsize": 219,
  "weight": 874,
  "locktime": 0,
  "vin": [
    {
      "txid": "6b71247fe48d11a1d9ff1e3ccc31417e6afdc6821382d5efc62ec964c267078f",
      "vout": 1,
      "scriptSig": {
        "asm": "00208e03bd637ebfec049d19df78d90085141eacbd14034793155f3822fd58d3dc5e",
        "hex": "2200208e03bd637ebfec049d19df78d90085141eacbd14034793155f3822fd58d3dc5e"
      },
      "txinwitness": [
        "",
        "304402202bee2dad46d2e21e159970e882c02db984b19d000a3794f16f54ef147befe66a02202f0c79a63679b98b900473a9e8d312c7958206a36ba2716c5b73ef5f8caf28bb01",
        "30440220770c2862744dee91fe0a487f47c92e9961c0b460ad1003e39f6f9f949bc11250022077ac039ded2ed13b07224f34e80716a3c0ee3fe8ef97012a1172728a62fe956201",
        "30440220466e6fbaaee25d2d88e841b5f40c4549380611d0c12b6a85920c241e3c1cd0f0022047d5b360717a5b210490ac85d8c1e72e257c7ffd6bad033a0229e41a0692de4701",
        "53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155ae"
      ],
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.19997810,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 b6c7733e23a271e57ca16e5cd885174917968db9 OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a914b6c7733e23a271e57ca16e5cd885174917968db988ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99"
        ]
      }
    }
  ]
}
