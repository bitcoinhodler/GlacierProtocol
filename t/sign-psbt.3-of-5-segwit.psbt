cHNidP8BAFUCAAAAATdD+0LzEYZ7MXq+HAbkcJnGpa9MtIruqDNFDCTrZLxrAQAAAAD/////AXIkMQEAAAAAGXapFLbHcz4jonHlfKFuXNiFF0kXlo25iKwAAAAAAAEBIAAtMQEAAAAAF6kUJSSn4pMppja/TB2N6g3GoIfl2RaHAQQiACCOA71jfr/sBJ0Z33jZAIUUHqy9FANHkxVfOCL9WNPcXgEFrVMhAyCElgfu0mXAn1HNoOlyQmVYJ2OEhwZRJQqk6GOMM0ztIQJ2bHBHKoo951HRJu3ayENjOPhEexlntao7EqK5pucS6SEDkIDdxEjn/kmFZ2p9LWVn38L+u4IHpvKblzcjP0w7RNYhA/otLAdlOvsKczQO6eaxD/DUYkyTsBEn6aae++1IRo4BIQKWabedfgXyDKeqQY7cg52pBEZyU8nkcZTLg+ae5BgxUVWuAAA=
