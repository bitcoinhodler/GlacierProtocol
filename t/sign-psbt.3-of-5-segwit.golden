Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.20000000 BTC in unspent inputs from cold storage address 2MvdcuGhBWhwiHjTr7XUSydtv5FmMge93jS
0.19997810 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
Fee amount: 0.00002190
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 3 required to complete transaction)?
#: Key #1: Key #2: Key #3: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.20000000 BTC in unspent inputs from cold storage address 2MvdcuGhBWhwiHjTr7XUSydtv5FmMge93jS
0.19997810 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
Fee amount: 0.00002190

Signing with private keys: 
cN5pwP2oCxokXYFdRVYczUqLqdA8GYqGWfRjaprrQXgHrS6PTHaD
cVyhd5Ei9HX2wSny8S2nYYtW2ubiUPhzdx1JUiSTjNicQJYTGLQV
cUv81BVz1JWuA9uwECPvfsoyA1TqmFJBJaQUPYrxkdf44zQSiTZY


Confirm? (y/n): 
Calculating transaction...

Final fee rate: 10.00 satoshis per vbyte

Raw signed transaction (hex):
020000000001013743fb42f311867b317abe1c06e47099c6a5af4cb48aeea833450c24eb64bc6b01000000232200208e03bd637ebfec049d19df78d90085141eacbd14034793155f3822fd58d3dc5effffffff0172243101000000001976a914b6c7733e23a271e57ca16e5cd885174917968db988ac0500473044022008a2d74f7f91344ca77ea01f772e9b5611b06a5bcc56da9f03e792fdee4ed83302207a168a4bf1bfc9485166ccfb25c8f6b36ba67120644586b8969c693b38fa02aa0147304402202655bb868bdbc2535ebcd08de758e1876b6599ce4bc7665a1769518a38278f6e0220771d25618baeef10c7d6650267e1f65e92da3bd99ba4cb12b9923edcb190c3500147304402203254e2c9be2e2595891f7130cdf403fee3db7d8a4bf373df8938f5b745ca4ca302207d1167b67df988478a757f08e9eb173a4795d53f8833838fbec9bb0075cc4e3701ad53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155ae00000000

Transaction fingerprint (md5):
77a56bf3a395c5966f18ccee3c4cd368
QR code for transaction written to transaction.png
